tools/src/main/scala/scala/scalanative/interflow/UseDef.scala

```diff
@@ -49,6 +49,9 @@ object UseDef {
       }
       super.onNext(next)
     }
+
+    override def onType(ty: Type): Type = ty
```

Comment:
Can you please add a small comment why are we're overriding `onType` here. It should be easier to follow up the code changes in the future.

Reply:
No problem